languages/python/exercises/concept/strings/.docs/after.md

```diff
@@ -0,0 +1,10 @@
+# After
+
```

Comment:
We've recently (as in yesterday) started to experiment with changing the `after.md` document a bit. The most important change is that the `after.md` document should now summarize what the exercise aimed to teach the student, like a recap of sorts. It can also still include references to interesting other resources like the ones you list here.

For an example of new `after.md` files, see https://github.com/exercism/v3/blob/master/languages/ruby/exercises/concept/strings/.docs/after.md and https://github.com/exercism/v3/pull/767/files